src/test/store/profile-view.test.js

```diff
@@ -259,6 +259,20 @@ describe('getJankMarkersForHeader', function() {
     expect(jankInstances.length).toEqual(1);
     expect(getJankInstantDuration(jankInstances[0])).toEqual(breakingPoint);
   });
+
+  it('will show BHR markers when there are no Jank markers present', function() {
+    const profile = getProfileWithMarkers([
+      ['a', 0, null, { type: 'BHR-detected hang' }],
```

Comment:
To be able to give this marker a "duration", we need `start` and `end` values. In the second item, we have `0` which is the start, but the end value is `null`. We should also add an end value, so we can see the duration of the marker. You can simply give `10` for example.